src/spec/helpers/functions_helper_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe FunctionsHelper do
+  describe '#render_markdown' do
+    context 'output does not contain headers' do
+      it { expect(helper.render_markdown('# Header')).to_not include('<h1>') }
+      it { expect(helper.render_markdown('## Header')).to_not include('<h2>') }
+      it { expect(helper.render_markdown('### Header')).to_not include('<h3>') }
+    end
+
+    context 'output does not contain images' do
+      it { expect(helper.render_markdown('<img src=\'https://example.com\' title=\'title\'/>')).to_not include('<img') }
+    end
+
+    context 'output does not contain any unescaped html tags' do
+      it { expect(helper.render_markdown('<del onclick="alert()">text</del>')).to_not include('onclick="alert()"') }
+      it { expect(helper.render_markdown('<abbr> Click Me </abbr>')).to_not include('<abbr>') }
```

Comment:
all of the above are the negative cases were we make sure harmful and unwanted tags/attributes are not passed to the output.

we'll need more tests for the positive cases that this function must render